### ISSUE TITLE
[Bug Report]: blue bar is always going back and forth even though I'm not doing anything 

### ISSUE THREAD

**Issue by SheepyVN** [WitherHosting/Bug-Tracker#10]
### Expected Behaviour

the blue bar should have stopped when I was on a fixed page

### Final Outcome

going back and forth

### Steps to Reproduce Issue

```text
Go to hosting panel and wait for second, notice under the "search" bar
```

### Screenshots

![image](https://github.com/WitherHosting/Bug-Tracker/assets/92708299/741cda1b-ccef-4a7b-9560-a064d464d124)

**Comment by RediPanda** [comment]
@SheepyVN This is to be expected as we now maintain a live alert system which polls every 30 seconds. We might take a look into polling either every minute or turn off the bar per poll. 

The page also self-hydrates itself with fresh data depending on the page location like backups, that way when you tab back into the page after creating a backup you'll instantly get the last fresh status of the backup.

**Comment by RediPanda** [comment]
Upon further inspection, it seems the fetching is inconsistent and seems to worsen over time. I'll take a look into the matter.

**Comment by RediPanda** [comment]
This should be fixed in (3.0.3) [d8636d0](https://github.com/WitherHosting/WitherPanel-v3/commit/d8636d0b6bdd27ab484aef3e0a78db41ab2805ec)
